### PR TITLE
Capstone 1: Global Summer Music Camp for Teenagers.

### PR DESCRIPTION
**Pull request by Kingjosh007**
This is my first capstone project in my journey at Microverse. I was expected to create a home page and an about page close to those in [this Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) by [Cindy Shin](https://www.behance.net/adagio07). Since I had to personalize the content of the website, I imagined it a random summer music camp for teenagers. It is all fictional. 

Here is an overview of what I did, regarding the project requirements.

1. I created the HTML markup for both pages, including links between the pages.
2. I styled both pages targetting 2 different types of screen (using media queries) : **mobile** (up to 768px wide) and **desktop** (wider than 768px).
3. I added some dynamic content using Javascript
4. I included mobile menu interactions and animations using Javascript.
5. I had to compromise in some places, to make the design fit my theme. That's why in some places the site doesn't look perfectly like the Behance template. That said, I tried to stick to the design wherever possible.
6. I used Sass for the styling. It is my first time using it in a project.
7. I wrote a nice-looking README file, including gifs of the website (both mobile and desktop versions).
8. I recorded a 5-minute long video showing me presenting my work. The video is available [here](https://www.loom.com/share/c27dba034466496c8f9a514cf4b26235) (updated).

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            <nav class="desktop-menu">
+                <div class="logo-container">
+                    <a href="index.html"><img src="images/logo.png" alt="Logo of the event organizers"></a>
+                </div>
+                <ul>
+                    <li class="links-container"><a href="index.html">Home</a></li>
+                    <li class="links-container"><a href="about.html" class="active">About</a></li>
+                </ul>
+            </nav>
+        </div>
```
Comment (henatan99) [inline about.html]: - [ ] The navbar in the desktop view is divided in a top dark bar and a bottom white bar. As you can see from the screenshot below there is a top dark bar with some links. Kindly make changes accordingly. :pray: 

image

File: about.html [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+                    class="fab fa-github"></i></a>
+            <a href="https://www.linkedin.com/in/king-josaphat-chewa-aa154011b" target="_blank"
+                rel="noreferrer noopener"><i class="fab fa-linkedin"></i></a>
+            <a href="https://angel.co/u/king-josaphat-nkouete-chewa" target="_blank" rel="noreferrer noopener"><i
+                    class="fab fa-angellist"></i></a>
+            <a href="https://twitter.com/KingJoChewa" target="_blank" rel="noreferrer noopener"><i
+                    class="fab fa-twitter"></i></a>
+            <a href="https://medium.com/@kingjochewa2007" target="_blank" rel="noreferrer noopener"><i
+                    class="fab fa-medium"></i></a>
+        </div>
```
Comment (henatan99) [inline about.html]: - [ ] The footer should contain a logo aligned to the left and text aligned to the right. Your design missed the logo. Always make sure you stick with the given design as it represents business/client requirements. Kindly compare your design with the original and make changes. 
- [ ] In addition, the footer has a dark background with white text for desktop view at least for the About page. 

image


image

File: css/home.css [hunk 3]
```diff
@@ -0,0 +1,968 @@
...
+  align-items: center;
+  gap: 1em;
+}
+
+.instructors .instructor {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 7px;
+}
```
Comment (henatan99) [inline css/home.css]: - [ ] In the instructor's section, the article's photos are aligned in the original design. But there is a misalignment in your design. Kindly make changes here. :clap: 
- [ ] In addition, the font for the description text is not the same as the original design. Please make sureto use the original typography. 

image

image

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @Kingjosh007 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 1 above)
Comment (Kingjosh007) [reply about.html]: Sorry, I had to be absent from Thursday to Saturday last week. So I'm repeating the week after having used only a single attempt. 
That change is done now.

File: about.html (hunk 2 above)
Comment (Kingjosh007) [reply about.html]: Sorry, I had to be absent from Thursday to Saturday last week. So I'm repeating the week after having used only a single attempt.
That change is done now.

File: css/home.css (hunk 3 above)
Comment (Kingjosh007) [reply css/home.css]: Sorry, I had to be absent from Thursday to Saturday last week. So I'm repeating the week after having used only a single attempt.
Those changes are done now.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Kingjosh007  🔥 

You worked hard in all the instructions that the previous Code Reviewer give you, now your Capstone project is complete! Good work!👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**